meta.go

```diff
@@ -905,6 +943,47 @@ func parseDevices(devices interface{}) map[string]Device {
 	return result
 }
 
+func parseDeployment(deployment interface{}, charmSeries []string, storage map[string]Storage) (*Deployment, error) {
+	if deployment == nil {
+		return nil, nil
+	}
+	if len(charmSeries) == 0 {
+		return nil, errors.New("charm with deployment metadata must declare at least one series")
+	}
+	deploymentMap := deployment.(map[string]interface{})
```

Comment:
!ok return error?

Reply:
no need, that validation is done by the schema definition

Reply:
ah, ic, thanks